src/js/events/ConfigActions.js

```diff
@@ -22,6 +21,24 @@ const ConfigActions = {
         });
       }
     });
+  },
+
+  fetchCCID: function () {
+    RequestUtil.json({
+      url: `${Config.rootUrl}/navstar/lashup/key`,
+      success: function (response) {
+        AppDispatcher.handleServerAction({
+          type: ActionTypes.REQUEST_CLUSTER_CCID_SUCCESS,
+          data: response
+        });
+      },
+      error: function (e) {
+        AppDispatcher.handleServerAction({
+          type: ActionTypes.REQUEST_CLUSTER_CCID_ERROR,
+          data: e.message
```

Comment:
do we need to do the whole `RequestUtil.getErrorFromXHR(xhr)` dance here? or is this cool?